deploy/kubernetes/Jenkinsfile.yml

```diff
@@ -18,5 +18,5 @@ stages:
             make check-no-changes;
         stage: checks
     k8sTest:
-        k8sTest: "v1.31.0,v1.30.4,v1.29.8,v1.28.13"
+        k8sTest: "v1.32.0,v1.31.0,v1.30.4,v1.29.8,v1.28.13"
```

Comment:
Are we still using jenkings?

Reply:
I plan to merge https://github.com/elastic/beats/pull/41625 so no mor ejenkins references around